test/lrud.test.js

```diff
@@ -648,4 +648,143 @@ describe('Given an instance of Lrud', () => {
       expect(toJSON(navigation.nodes.root.children)).toEqual(['child-c', 'child-d'])
     })
   })
+
+  describe('handleKeyEvent - with overrides', () => {
+    it('should move through a horizontal list as expected [`override`]', () => {
+      navigation.overrides = {
+        'override-1': {
+          id: 'child2',
+          direction: 'RIGHT',
+          target: 'child1'
+        }
+      }
+      navigation.register('root', { orientation: 'horizontal' })
+      navigation.register('child1', { parent: 'root', selectAction: true })
+      navigation.register('child2', { parent: 'root', selectAction: true })
+      navigation.register('child3', { parent: 'root', selectAction: true })
+      navigation.register('child4', { parent: 'root', selectAction: true })
+
+      navigation.currentFocus = 'child2'
+      navigation.handleKeyEvent({ keyCode: 39, stopPropagation: noop })
+
+      expect(navigation.currentFocus).toEqual('child1')
```

Comment:
Isn't the `activeChild` affected ?

Reply:
It should be OK, but will add some more test cases 👍

Reply:
Added a ton of `expects`s around `activeChild` values

https://github.com/bbc/lrud/pull/4/commits/d3497222eda33e136f382a64f7b6673c275b2f38

👍